cengage/react-magma
# Fix - Updates to Expressive text styles

Issue: # [1156](https://github.com/cengage/react-magma/issues/1156)

## What I did
- Updated the Expressive text variant: font sizing, weights, and line-heights

## Checklist 
- [N/A] I have added tests that prove my fix is effective or that my feature works

## How to test
- In Docs > Design > Typography, ensure the Expressive Body Text Styles are correct
- In Docs > Design > Typography, ensure the Expressive Heading Text Styles are correct
